src/client/components/MealList.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import RingLoader from "react-spinners/RingLoader"; //npm install --save react-spinners
+import MealsCSS from "./UI/meals.module.css";
+import Meal from "./Meal";
+
+const MealList = () => {
+  const [listOfMeals, setListOfMeals] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  // Loading spinner
+
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
```

Comment:
Probably this is an experimentation, which is good 👍🏽 
It would be better to attach the loader behaviour to what happens with data fetching in the useEffect 💡

Reply:
I just wanted to show a cool spinner when the main page loads, so I decided to use setTimeout(). But I'll move it to another place to make it more logical :)

Reply:
It totally makes sense to show off the spinner! 
So up to you anyway 👍